pkg/csi/service/node.go

```diff
@@ -701,7 +701,14 @@ func (s *service) NodeGetInfo(
 				return nil, status.Errorf(codes.Internal, err.Error())
 			}
 		}
-		zone, region, err := nodeVM.GetZoneRegion(ctx, cfg.Labels.Zone, cfg.Labels.Region)
+		tagManager, err := cnsvsphere.GetTagManager(ctx, vcenter)
```

Comment:
Why are we not checking for err != nil after this line ?

Reply:
yes we need to check err !=nil, let me fix that

Reply:
done